### DISCUSSION TITLE
Can GitHub Apps/Bots edit wikis?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#24835] [Apps, API and Webhooks]
<p>I wanted to ask if GitHub apps (or bots, depending on how you look at it) can edit wikis of a repository.</p>
<p>I had an idea for an app/bot, but it would require it to have access to the wiki(s) of the repositories it gets access to. Unfortunately, am I unable to find any topic or documentation about this.</p>
<p>So I would appreciate if someone could lead me to the right place.</p>

**Comment by xmo-odoo** [comment, marked as answer]
<p>Apps or bots do, to an extent, have the ability to edit wikis: while there is no API endpoint, the wiki is available as a git repository (the wiki for user/project.git is user/project.wiki.git). So a bot can just check that out, create new commits / modifications, and push the changes back (if they have push access to the repo I guess).</p>
